webpack-dll.config.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External Dependencies
+ */
+const path = require( 'path' );
+const webpack = require( 'webpack' );
+
+/**
+ * Internal Dependencies
+ */
+const config = require( './server/config' );
+
+const bundleEnv = config( 'env' );
+
+module.exports = {
+	entry: {
+		vendor: [ path.join( __dirname, 'client', 'vendor-dll.js' ) ]
```

Comment:
Couldn't we pass all the vendor libs directly in the array here, instead of as a separate file in `client`? Basing this assumption [on the current configuration](https://github.com/Automattic/wp-calypso/blob/f285c47e26d3aa3e9093a0d8f98b4b84ddc39cb0/webpack.config.js#L98).